src/main/java/games/strategy/triplea/ui/MovePanel.java

```diff
@@ -803,19 +805,19 @@ private void selectUnitsToMove(final List<Unit> units, final Territory t, final
       // add all
       if (me.isShiftDown()) {
         // prevent units of multiple owners from being chosen in edit mode
-        final Match.CompositeBuilder<Unit> ownedNotFactoryBuilder = Match.newCompositeBuilder();
+        final PredicateBuilder<Unit> ownedNotFactory = PredicateBuilder.of(Matches.always());
```

Comment:
Would it be worth having an initial builder method that creates a "true" predicate? Something like PredicateBuilder.always() or PredicateBuilder.true()?